EIPS/eip-5496.md

```diff
@@ -13,15 +13,15 @@ requires: 721
 
 
 ## Abstract
-This specification defines standard functions that extends current [EIP-721](./eip-721.md) standard providing sharable multiple privileges for a NFT collection. The privileges could be on-chain ones, such as voting rights, or to claim a targeted airdrop; or off-chain ones, such as a coupon in Amazon, or a discount in a local New York restaurant, or even a VIP lounge access in the airport. There could be multiple privileges attached to one NFT and each privilege holder of an NFT can transfer the privilege to others. The ownership of the privileges is verifiable. On top of the multiple-privilege extension, this standard also contains methods to give the provider to specify the privilege non-sharable or sharable. Sharable privileges can be done by simply cloning a privilege, furthermore, the provider would be able to adjust the details of privileges according to the spreading path. Expiration period can also be set to each privilege. The proposal depends on and extends the existing [EIP-721](./eip-721.md).
+This EIP defines an interface extending [EIP-721](./eip-721.md) to provide shareable multi-privileges for NFTs. Privileges may be on-chain (voting rights, permission to claim an airdrop) or off-chain (a coupon for an online store, a discount at a local restaurant, access to VIP lounges in airports). Each NFT may contain many privileges, and the holder of a privilege can verifiably transfer that privilege to others. Privileges may be non-shareable or shareable. Shareable privileges can be cloned, with the provider able to adjust the details according to the spreading path. Expiration periods can also be set for each privilege.
```

Comment:
My first question would be, would it need to restrict to 721 or does it also work for EIP-1155? It would help adoption to support both but also up to you.

Reply:
We will not support EIP-1155.
In EIP-5496 each privilege links to one token id of EIP-721, due to EIP-721 is non-fungible and unique. It's difficult to manage privilege for each token id in EIP-1155 which may contain multiple fungible NFTs with the same id.